series/series.go

```diff
@@ -82,7 +82,7 @@ func (e boolElements) Len() int           { return len(e) }
 func (e boolElements) Elem(i int) Element { return &e[i] }
 
 // ElementValue represents the value that can be used for marshaling or
-// unmarshaling Elements.
+// un-marshaling Elements.
```

Comment:
The previous spelling was correct: https://en.wikipedia.org/wiki/Unmarshalling

Reply:
And I can address this and will target development branch. I am guessing the intent for master is to be fairly stable.

Thanks a lot!